client/src/pages/EditProfile/EditProfile.tsx

```diff
@@ -0,0 +1,198 @@
+import { useState, Fragment } from 'react';
+import useStyles from './useStyles';
+import { Typography, Grid, TextField, MenuItem, Select, Box, FormControl, Paper, InputLabel } from '@material-ui/core';
+import Button from '@material-ui/core/Button';
+import { KeyboardDatePicker, MuiPickersUtilsProvider } from '@material-ui/pickers';
+import DateFnsUtils from '@date-io/date-fns';
+
+import { editProfile } from '../../helpers/APICalls/profile';
+
+import { useSnackBar } from '../../context/useSnackbarContext';
+import { useAuth } from '../../context/useAuthContext';
+
+export default function EditProfile(): JSX.Element {
+  const classes = useStyles();
+  const { updateSnackBarMessage } = useSnackBar();
+
+  const [firstName, setFirstName] = useState<string>('');
```

Comment:
Why don't we consider using 'Formik' for a better managed state and form validation? :)